### DISCUSSION TITLE
Removing intro.md from articles breaks the config

### DISCUSSION THREAD

**Discussion by RedX2501** [dotnet/docfx#8270] [General]
**Operating System**: `Windows`

**DocFX Version Used**:  2.56.1

**Template used**: `statictoc`

**Steps to Reproduce**:

Create a default config.
Add an article to the `articles` folder.
Add the new article to the `articles/toc.yml`
Remove `intro.md` from the `articles/toc.yml`
Remove `intro.md`
Generate website

**Expected Behavior**:
Uses the first available article as start location

**Actual Behavior**:
Tries to find index.html which no longer exists and displays an empty page.

**Comment by KalleOlaviNiemitalo** [comment]
I can't reproduce this using the reported steps.

If I do not run `docfx build` before removing the `intro.md` link from `articles/toc.yml`, then there is no link to `articles/intro.html` or `articles/index.html` anywhere in `_site`.

If I run `docfx build` before removing the `intro.md` link from `articles/toc.yml`, and again after removing the `articles/intro.md` file, then `_site/articles/intro.html` still exists.
